docs/maatregelen/0-org-08-governance_per_risicocategorie.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Richt algoritmegovernance in per risicocategorie
+toelichting: Richt algoritmegovernance in zodat per risicocategorie de juiste processtappen moeten worden doorlopen
+levenscyclus:
+- organisatieverantwoordelijkheden
+onderwerp:
+- governance
+rollen:
+- projectleider
+# - beleid-en-advies
+
+---
+
+## Maatregel
+
+Richt algoritmegovernance in zodat per risicocategorie de juiste processtappen moeten worden doorlopen
+
+## Toelichting
+
+Er is verschil in de vereisten en maatregelen die nodig zijn afhankelijk van het type risico en impact. 
+Zo vraagt bijvoorbeeld de AI-verordening om meer vereisten bij hoog-risico AI-systemen zijn. 
+Een meer uitgebreide governance is daar nodig, in de zin dat er meer handelingen en verslagleggingen nodig zijn.
+Een ander voorbeeld is dat voor generatieve AI (AI voor algemene doeleinden in de AI-verordening) er meer vereisten zijn.
+
+Afgeleid van de term "minimum viable product" wordt de processen van compliance doen aansluiten op wat nodig is ook wel "minimum viable compliance" genoemd.
```

Comment:
Dit komt uit de lucht vallen en lijkt me weinig toegevoegde waarde hebben of het moet beter omschreven worden.